<launch>
  <arg name="control_cmd_timeout_sec" default="0.5"/>
  <arg name="loop_rate" default="50.0"/>
  <!-- <arg name="" default=""/> -->

  <!-- for test -->
  <include file="$(find-pkg-share global_parameter_loader)/launch/global_params.launch.py">
    <arg name="vehicle_model" value="whill"/>
  </include>

  <node pkg="autoware_ros2_whill_adapter" name="autoware_ros2_whill_receiver" exec="autoware_ros2_whill_receiver" output="log">
  </node>

  <node pkg="autoware_ros2_whill_adapter" name="autoware_ros2_whill_sender" exec="autoware_ros2_whill_sender" output="log">
    <param name="control_cmd_timeout_sec" value="$(var control_cmd_timeout_sec)"/>
    <param name="loop_rate" value="$(var loop_rate)"/>
  </node>

</launch>
